### DISCUSSION TITLE
How to get reproducible collection setups and changes?

### DISCUSSION THREAD

**Discussion by kassuro** [appwrite/appwrite#3043] [Q&A] [answered]
So I am in the middle of building a small App with appwrite as the main backend.

While I'm getting near the point of testing the app in a staging environment I find myself more often thinking about how am I supposed to bring changes of collections or the general setup from one environment to another?

I have tried build some scripts with the node server SDK but the results aren't that good. 
For now I have a setup and cleanup script and already have problems with collections not being removed correctly and such stuff.

So how am I supposed to handle this? The docs don't really have anything about this topic (or I'm to stupid to find it?)

**Comment by gewenyu99** [comment, marked as answer]
:wave: You can use the CLI to accomplish this. You can setup an Appwrite project and use the CLI to fetch existing collections into `appwrite.json` which will save collection schemas. You can then point the CLI to a different project/Appwrite instance, which will create the same set of collections on the new instance.

https://appwrite.io/docs/command-line#deployCollections

**Reply by kassuro** [reply]
Oh wow, how could I miss that. 🤦‍♂️
Thanks for your answer. This will help a lot.

**Reply by gewenyu99** [reply]
This is new in 0.13 :') I learned about this last week!

**Reply by sanderkollerie** [reply]
How does this work for you? If you use this option the CLI deletes all the existing data. So if you create something new in test and then want to deploy to live, this option doesn't work. All existing data in your live environment is gone.
